src/app.types.tsx

```diff
@@ -272,3 +272,42 @@ export interface SessionListItem {
   timestamp: string;
   _id: string;
 }
+
+export interface FunctionTag {
+  type: 'channel' | 'functionToken' | 'function' | 'number' | 'string';
+  value: string;
+  label: string;
+}
+
+export interface FunctionToken {
```

Comment:
I wonder in naming wise, to keep consistency with filters this should be `FunctionOperator`, and thus `FunctionTag` can be `FunctionToken` (which is more equivalent to the `Token` type used in filters).